tests/functional/pv/pvc_snapshot/test_rbd_cbt_metadata.py

```
@@ -0,0 +1,423 @@
+        self.pod_factory = pod_factory
+        self.snapshot_factory = snapshot_factory
+        self.teardown_factory = teardown_factory
+
+        self.cbt_runner = VerifierTool(self.namespace)
+        self.cbt_runner.setup()
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request):
+        """Clean up CBT runner."""
+
+        def finalizer():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `finalizer`

Add return type annotation: `None`

(ANN202)